frontend/app/templates/components/transportation/tdf/vehicle-occupancy.hbs

```diff
@@ -51,24 +51,24 @@
         </td>
 
         {{#if factor.temporalVehicleOccupancy}}
-          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.am}}</td>
-          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.md}}</td>
-          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.pm}}</td>
-          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.saturday}}</td>
+          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.am input=(action "setIsEditing")}}</td>
+          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.md input=(action "setIsEditing")}}</td>
+          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.pm input=(action "setIsEditing")}}</td>
+          <td>{{input type="number" step=".01" min="0" value=vehicleOccupancy.auto.saturday input=(action "setIsEditing")}}</td>
```

Comment:
I think the preferred way to pass closure actions is to not use the string, but the direct prop, like this:

`input=(action this.setIsEditing)`